RichClipboard.package/RCClippingsList.class/properties.json

```diff
@@ -11,7 +11,8 @@
 		"regexFilter",
 		"isInPrivateMode",
 		"contextMenuIsOpen",
-		"sortBlock" ],
+		"sortBlock",
+		"timeoutMS" ],
```

Comment:
Warum gibt es diese Variable? Wir benutzen doch sowieso immer nur das default Timeout. Und ob wir uns in der Testinstanz befinden können wir mit RichClipboard>>isTestInstance überprüfen (und dann dementsprechend das Timeout kleiner wählen).

Bitte auch getter und setter entfernen. (Eventuell macht es Sinn, den Getter zu behalten und dort die Unterscheidung zwischen Test und nicht Test einzubauen. Dann muss das nicht dahin wo es angewendet wird)

Reply:
Ich sehe grade, dass ihr gar keine Tests habt. Von daher ist diese Variable ja sowieso unnötig